composer.json

```diff
@@ -126,6 +126,7 @@
         "symfony/twig-bundle": "^5.4 || ^6.0",
         "symfony/validator": "^5.4 || ^6.0",
         "symfony/webpack-encore-bundle": "^1.15",
+        "symfony/workflow": "^5.4 || ^6.0",
```

Comment:
I'm wondering if it should be hard dependency

Reply:
My idea was to keep the two dependencies as required packages, and deprecate the usage of winzou and remove it on Sylius 2.0. Moving winzou on dev requirements should be annoying for migrations to 1.13. I think it's a lot easier here to add it on prod requirements if it'll be the future package.

Reply:
Let's make it hard dependency then 👍

Reply:
Is there a discussion somewhere about removing winzou/sm package(s)? I'm interested in motivations of this change